tools/cmdline.txt

```diff
@@ -0,0 +1 @@
+dwc_otg.lpm_enable=0 console=serial0,115200 console=tty1 root=PARTUUID=1b4801d2-02 rootfstype=ext4 elevator=deadline fsck.repair=yes rootwait ip=
```

Comment:
I would rather append `ip=x.x.x.x` instead of just `x.x.x.x`. This way the file will always be correctly formed, even before it is edited, and it might help with the dhcp configuration later.

Reply:
Done.